.gitignore

```diff
@@ -1,4 +1,5 @@
 .git/
 .idea/
 target/
-*.iml
\ No newline at end of file
+*.iml
+Driver_Notes/
```

Comment:
Is it possible to remove Driver_Notes folder? I thought we can download drivers without any additional files, just exe files

Reply:
Yes, it was downloaded with driver by default, so I left it.
Please see deletion in the following commit.